### ISSUE TITLE
</br> tag on strings

### ISSUE THREAD

**Issue by horus68** [salesagility/SuiteCRM#1770]
#### Issue

`</br>`  is used 12 times on different code files.
See pic:
![brtag](https://cloud.githubusercontent.com/assets/3055443/17017031/92d097d6-4f29-11e6-8be7-cf33151d2fe3.png)
#### Possible Fix

`<br> or <br/> or <br />` 
Just pick one!
It would be useful to have a standard coding style on all SuiteCRM code files.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [</br> tag on strings](https://trello.com/c/0egLm4CC/85-br-tag-on-strings)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
